index.php

```diff
@@ -0,0 +1,5 @@
+<?php 
+
+include_once("pages/home.php");
```

Comment:
if this page only includes home.php, put the content of home into index and delete the empty home file